assets/prefabs/recipe/cook/RecipeToast.prefab

```diff
@@ -0,0 +1,19 @@
+{
+    "ProcessDefinition": {
+        "processType": "Cooking:Cooking"
+    },
+    "CraftingStationRecipe": {
+        "recipeId": "Hot food|Cooking:Toast",
+
+        "recipeComponents": ["1*Bread"],

```

Comment:
You *may* need to qualify this `"Bread"` as `"Cooking:Bread"` however I cannot get the recipe to work either way. That is probably more of a problem with Cooking or the workspace than your recipe, it looks fine.

Did you try to test it? Probably the recipe won't work but at least `give toast` does give a valid item so that's good :-)